### ISSUE TITLE
Ensure that clickable elements on the page are tabbable

### ISSUE THREAD

**Issue by teenie-quaggard** [FACxBeamery/Tom_Thomas_7_through_9#12]
At the moment, I cannot tab onto any individual element on the page. Please change this for accessibility purposes!
